src/assets/sass/protocol/templates/_card-layout.scss

```diff
@@ -158,6 +158,23 @@
     }
 }
 
+/* -------------------------------------------------------------------------- */
+// 3 picto card layout
+// On small screens cards stack vertically and span full container width.
+// On medium screens cards span 1 third width.
+.mzp-l-card-third {
+    @media #{$mq-md} {
+        @include clearfix;
+
+        .mzp-c-card-picto {
+            width: 33%;
```

Comment:
It's a minor thing but we should try and use `calc(100% / 3)` instead of 33% to keep widths from becoming fractions of pixels and keep the layout consistent.

At the moment we are using the Bedrock's legacy breakpoint/ max-width so we won't be able to completely avoid fractions of pixels, but I'm hoping to adjust those values when we have almost everything running protocol.

There are also a few places where we are using 33.3% (_card-layout.scss).

Reply:
I believe browsers generally round down decimal CSS values when it comes to rendering, so I think it's probably no different to use `33.3%` without `calc()` here, rather than  ending up with a value like `33.3333333%`.